frontend-service/src/_styles/_cssVariables.scss

```diff
@@ -15,7 +15,7 @@
   --c-yellow-50: #ffe879;
   --c-yellow-25: #fff4bb;
   --c-black-color: #000000;
-  --c-gray-110: #404040;
+  --c-gray-110: #404041;
```

Comment:
@alfaredu I have made this slight change in order to write a test which will catch that